addon/components/rdf-input-fields/concept-scheme-radio-buttons/show.hbs

```diff
@@ -0,0 +1,16 @@
+<AuLabel class="{{if this.errors "au-c-label--error"}}" for={{this.inputId}}>
+  {{@field.label}}
+  {{#if this.isRequired}}
+    <AuPill>Verplicht</AuPill>
+  {{/if}}
+</AuLabel>
+
+{{#each this.options as |option|}}
+  <label class="radio">
+    <input type="radio" class="radio__toggle" disabled={{true}} checked={{if (eq option.label option.selected.label) "true" "false"}} name={{@field.uri}} value={{option}}>
```

Comment:
Does this check work? I never seem to find where you put the selected on an `option`, within the `options`? Also, wouldn't it be better to mach on the `subject` instead off the `label`?

Reply:
Yes this actually fixes the  issue I had. Will push in a minute